doc/src/manual/profile.md

```diff
@@ -336,6 +336,23 @@ and how much garbage it collects each time. This can be enabled with
 [`GC.enable_logging(true)`](@ref), which causes Julia to log to stderr every time
 a garbage collection happens.
 
+### Allocation Profiler
+
+The allocation profiler records the stack trace, type, and size of each
```

Comment:
I think the documentation should mention that this doesn't actually record every allocation (i.e. not the ones coming from codegen-emitted calls to `jl_gc_pool_alloc`).

Reply:
Good point; I'll add something.

Btw, trying to at least grab stacks for those, if not types; but don't have it right yet: https://github.com/vilterp/julia/pull/14